share/mk/bsd.opts.mk

```diff
@@ -53,6 +53,7 @@ __<bsd.opts.mk>__:
 
 __DEFAULT_YES_OPTIONS = \
     ASSERT_DEBUG \
+    CHERI_CAPREVOKE \
```

Comment:
You also probably want to add the relevant files to tools/build/options (WITH_CHERI_CAPREVOKE and WITHOUT_CHERI_CAPREVOKE perhaps) and then run 'make makeman` from the top-level to regenerate src.conf.5.  The regen would normally be done as a separate commit, and for CheriBSD I would drop the diff to the '.Dd' line in the regen to avoid future merge conflicts.

Reply:
I can't get `make makeman` to DTRT when cross-building. :(

Reply:
(Also, I or Brooks could also do this post-merge as a fixup easily enough)